packages/common/src/types/database.ts

```diff
@@ -11,7 +11,7 @@ export type TDatabase = {
 };
 
 export type TDatabaseDetail = TDatabase & {
-  collection: TCollectionAndMetadata[];
+  collection: TMetadataCollection[];
```

Comment:
maybe consider changing the type name to `TCollectionMetadata` since it reads more naturally